.github/workflows/s3-mirror.yml

```diff
@@ -36,6 +36,11 @@ jobs:
         with:
           fetch-depth: 0
 
+      - name: Render README.md to HTML
+        run: |
+          pip install markdown
+          echo "<html><body>$(python -m markdown README.md)</body></html>" > README.html
```

Comment:
Hmm, why doesn't that markdown package does this, seems weird?

Reply:
Well, its primary use case is being a library not a command-line tool, so I guess you would normally just embed it in a page you already have. The library does have some command-line options but I could not see anything to add the document wrapper around it. I tried `mistune` too but it produced very similar output, without the wrapper (at least by default).

I didn't want to dig too deep into that because I expect that we'll just replace it completely when we add the system that automatically lists available binaries. But if you think it's worthwhile I could try some other libs. For example [markdown-html-action](https://github.com/ZacJW/markdown-html-action) does not add these on its own (it only adds `doctype`) so apparently the library it uses does it automatically.

Reply:
Well, its primary use case is being a library not a command-line tool, so I guess you would normally just embed it in a page you already have. The library does have some command-line options but I could not see anything to add the document wrapper around it. I tried `mistune` too but it produced very similar output, without the wrapper (at least by default).

I didn't want to dig too deep into that because I expect that we'll just replace it completely when we add the system that automatically lists available binaries. But if you think it's worthwhile I could try some other libs. For example [markdown-html-action](https://github.com/ZacJW/markdown-html-action) does not add these on its own (it only adds `doctype`) so apparently the library it uses does it automatically.

Reply:
The documentation (https://python-markdown.github.io/cli/) just mentions `echo "Some **Markdown** text." | python -m markdown > output.html`, without the need for extra html/body wrapping.

Reply:
Yeah. Maybe they should mention it more explicitly but I found https://github.com/Python-Markdown/markdown/issues/492 which says that it's by design and that a CLI option won't be added because it's out of scope. Unfortunately it's our responsibility to provide a template/wrapper when using it.

Reply:
Thanks! They could have just documented this on their usage guide, sigh.